sqlmesh/core/console.py

```diff
@@ -1509,10 +1511,17 @@ def show_model_difference_summary(
         added_snapshot_models = {s for s in added_snapshots if s.is_model}
         if added_snapshot_models:
             self._print("\n**Added Models:**")
-            for snapshot in sorted(added_snapshot_models):
-                self._print(
-                    f"- `{snapshot.display_name(environment_naming_info, default_catalog, dialect=self.dialect)}`"
-                )
+            added_models = sorted(added_snapshot_models)
+            list_length = len(added_models)
+            if not self.verbose and list_length > self.INDIRECTLY_MODIFIED_DISPLAY_THRESHOLD:
```

Comment:
Where do we set the level of verbosity? Can the `plan` command be run in a verbose mode?

Reply:
Yes `plan` command has the option to be executed in `verbose` mode. and it is then set in [/cli/main.py#L393](https://github.com/TobikoData/sqlmesh/blob/b46206bb54237f837c3b83fcbb4ae7c9e94b8bf0/sqlmesh/cli/main.py#L393)

Reply:
nice, thanks!